modules/catalog/src/main/scala/lucuma/catalog/csv/TargetImport.scala

```diff
@@ -191,15 +191,19 @@ object TargetImport:
     unitAbbv[T].foldLeft(
       Map.empty[String, Units Of Brightness[T]]
     ) { (map, unit) =>
-      val replaced = if (unit._1.contains("²") || unit._1.contains("Å")) {
+      val replaced  = if (unit._1.contains("²") || unit._1.contains("Å")) {
         Map(unit._1.replaceAll("²", "2").replaceAll("Å", "A")  -> unit._2,
             unit._1.replaceAll("²", "^2").replaceAll("Å", "A") -> unit._2
         )
       } else Map.empty
-      val angstrom = if (unit._1.contains("Å")) {
+      val angstrom  = if (unit._1.contains("Å")) {
         Map(unit._1.replaceAll("Å", "A") -> unit._2, unit._1.replaceAll("Å", "A") -> unit._2)
```

Comment:
Just realized: this seems to be attempting to add the same key/value twice to the map (which is not an issue since it's a map).

Also, if there's an `Å` and a `²`, do we want to allow `Å`/`2`, `Å`/`ˆ2` and `A`/`²`?